bigchaindb/core.py

```diff
@@ -9,7 +9,7 @@
 import sys
 
 from abci.application import BaseApplication
-from abci.types_pb2 import (
+from github.com.tendermint.tendermint.abci.types.types_pb2 import (
```

Comment:
nitpicky, but why are these imports so weird?

Reply:
good feedback. I cleaned up some difficulties with the given bigchaindb-abci. Therby exposing the types_bp2. I resolved this by proxy using abci as a proxy. similar to befoe.